server/lib/pbench/indexer.py

```diff
@@ -1089,14 +1129,31 @@ def _handle_iteration(self, iter_data, iter_name, iter_number, result_json):
                     bm_md[new_key] = val
             conflicts = new_conflicts
 
+        try:
+            curr_runtime = bm_md['runtime']
+        except KeyError:
+            # Ignore a missing 'runtime' field
+            pass
+        else:
+            # Ensure the "runtime" field is a string
+            bm_md['runtime'] = str(bm_md['runtime'])
+
         try:
             bm_md['uid_tmpl'] = bm_md['uid']
         except KeyError:
-            # Ignore a missing uid field
+            # Ignore a missing 'uid' field
             pass
         else:
             bm_md['uid'] = ResultData.expand_template(bm_md['uid_tmpl'], bm_md, run=self.run_metadata)
 
+        try:
+            bm_md['trafficgen_uid_tmpl'] = bm_md['trafficgen_uid']
+        except KeyError:
+            # Ignore a missing 'trafficgen_uid' field
+            pass
+        else:
+            bm_md['trafficgen_uid'] = ResultData.expand_template(bm_md['trafficgen_uid_tmpl'], bm_md, run=self.run_metadata)
+
```

Comment:
Well because we don't have an infrastructure for generic transformations.  I'd like to solve that via future Scribe integration work, but seems over-kill to do it here given this is only one UID field that needs to be fixed up.

Is that acceptable?

Reply:
I would appreciate at least a comment, explaining the problem.